### PR TITLE
Add html aggregated transformer and output

### PR DESCRIPTION
**Pull request by thiessenp-cds**
# Summary | Résumé

Updates the html aggregated file download. The api query was renamed from html-table to html-aggregated to fit more with the file aggregation functionality and a `lang=fr|en` param added to set the language output.  Also the html aggregated file was created and structured.

### CODE REVIEW

**Comment by timarney** [comment]
@thiessenp-cds if you need help fixing up the conflict reach out to @dsamojlenko

File: components/form-builder/app/responses/DownloadButton.tsx [hunk 1]
```diff
@@ -46,6 +46,33 @@ export const DownloadButton = ({
...
+        const anchorElement = document.createElement("a");
+        anchorElement.href = href;
+        anchorElement.download = fileName;
+        document.body.appendChild(anchorElement);
+        anchorElement.click();
+        document.body.removeChild(anchorElement);
+        window.URL.revokeObjectURL(href);
+      });
+
+      return; //TODO TEMP - when/where to download the aggregated file
```
Comment (dsamojlenko) [inline components/form-builder/app/responses/DownloadButton.tsx:73]: Can probably just leave off this wiring for now and just include the API response in this PR

File: lib/responseDownloadFormats/html-aggregated/components/AggregatedTable.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            </th>
+          ))}
+        </tr>
+      </thead>
+      <tbody>
+        {submissions.map((submission: Submission, index) => {
+          return (
+            <tr
+              key={submission.id}
+              className={`flex border-t-2 border-t-grey-300 ${
```
Comment (dsamojlenko) [inline lib/responseDownloadFormats/html-aggregated/components/AggregatedTable.tsx:43]: `border-t-grey-300` doesn't exist. Probably due to the british spelling (grey vs gray) ... also be sure to check the custom tailwind config to make sure the shade you want is there

File: lib/responseDownloadFormats/html-aggregated/components/ResponseHTMLAggregated.tsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  // Note: The "\r\n" delimiter may be OS dependent. If so use an actual newline with .join(`
+  // `)
+  const confirmationCodes = formResponseSubmissions.submissions
+    .map((submission) => submission.confirmationCode)
+    .join(`\r\n`);
+
+  const submissions = formResponseSubmissions.submissions as Submission[];
+
+  const headersForTable = [
+    { title: "Submission number / [fr]Submission number", type: "formData" },
```
Comment (dsamojlenko) [inline lib/responseDownloadFormats/html-aggregated/components/ResponseHTMLAggregated.tsx:39]: There are translations available for these strings in Figma

**Review by dsamojlenko — APPROVED** [review]

**Pull request merged**